src/index.js

```diff
@@ -0,0 +1,134 @@
+          <i class="fa-solid fa-ellipsis-vertical"></i><i class="fa-solid fa-pencil ocult"></i><i class="fa-regular fa-trash-can ocult"></i>
+        </li>
+        
+        `
+
+      } else if(task.completed === false ){
+        taskString = `
+        <li draggable='true' class='task container' id=${tasks.indexOf(task)}><input type="text" class="edit ocult"/><div class="input-container"><input type="checkbox" class='checkbox' id='check${tasks.indexOf(task)}'><label for='check${tasks.indexOf(task)}' class='label'>${task.description}</label></div>
+          <i class="fa-solid fa-ellipsis-vertical"></i><i class="fa-solid fa-pencil ocult"></i><i class="fa-regular fa-trash-can ocult"></i>
+        </li>
+        `
+      }
+
+    const taskHtml = parser.parseFromString(taskString, 'text/html').body.firstChild;
+    displayList.appendChild(taskHtml);
+    
+    //Add event to complete a task 
+
+    const checkbox = taskHtml.querySelector('.checkbox');
+   
+    completed(checkbox, task, tasks);
+
+    // Add event to select one task
+
+    const selectBtn = taskHtml.querySelector('.fa-ellipsis-vertical');
+    Task.selectTask(selectBtn);
+
+    //Add event to delete Task
+
+    const trashBtn = taskHtml.querySelector('.fa-trash-can');
+    Task.deleteTask(trashBtn);
+
+    // Add event to edit the task 
+    
+    const editBtn = taskHtml.querySelector('.fa-pencil');
+    const input = taskHtml.querySelector('.edit');
+    input.value = task.description;
+    Task.editTask(editBtn);
+    const label = taskHtml.querySelector('.label');
+    let clickCounter = 0;
+
+      editBtn.addEventListener('click',(e) => {
+        if(input.value.length > 0) {
+          label.textContent = `${input.value}`;
+          task.description = `${input.value}`;
+          localStorage.setItem('tasks', JSON.stringify(tasks));
+          e.target.parentElement.classList.remove('select');
+          clickCounter++; 
+          console.log(clickCounter);
+          if(clickCounter >= 2){
+            editBtn.classList.toggle('ocult');
+            trashBtn.classList.toggle('ocult');
+            selectBtn.classList.toggle('ocult');
+            clickCounter = 0;
+          }
+        }
+      }
+      )
+    })
+    changeTask()
+  }
+}
+
+DisplayList.showList();
+
+export { DisplayList }
+
+// Create event to add new task
+
+const addTask = document.getElementById('add-task');
+const submitBtn = document.querySelector('.submit-button');
+
+submitBtn.addEventListener('click', function(e) {
+  e.preventDefault();
+  if(addTask.value.length > 0) {
+    const tasks = LocalStorage.getTasks();
+    const task = new Task(addTask.value, tasks.length);
+    Task.addTask(task);
+    LocalStorage.addTask(task);
+    DisplayList.showList();
+    addTask.value = '';
+  }
+})
+
+
+
```

Comment:
kindly. Your code is perfect. The syntax was perfectly used. My recommendation is not to leave a lot of lines empty between the code. This will help you to have fewer lines and more orderly.